docs/src/theme/MDXComponents.js

```diff
@@ -23,4 +24,5 @@ export default {
   SideBySideContainer,
   TabItem,
   Tabs,
+  BuiltinSearch,
```

Comment:
Since this component is only used on one page. we can actually just import it in the file where it's needed. I'll add an example above.